pages/directory/index.tsx

```diff
@@ -28,14 +28,14 @@ const Directory: NextPage<{ users: any[] }> = props => {
 };
 
 Directory.getInitialProps = async ({ req }) => {
-  const axios = createAxiosInstance(req);
-  const response = await axios.get('/api/users');
-  const users = response.data.users;
-
   try {
+    const axios = createAxiosInstance(req);
+    const response = await axios.get('/api/users');
+    const users = response.data.users;
+
     return { users };
   } catch (error) {
-    return { users };
+    return error;
```

Comment:
This is why loosely typing things can be problematic...

Our `Directory` component is expecting to receive an object with the structure `{ users: any[] }`. If the API returns an error, the `catch` case is returning the `error` object instead of a structure the component is expecting.